rover_description/rover.xacro

```diff
@@ -139,6 +231,38 @@
     <origin xyz="${caster_wheel_joint_offset} 0 -${caster_wheel_radius+0.005}" rpy="0 0 0" />
   </joint>
 
+  <!--Camera Joint-->
+  <joint name="camera_joint" type="fixed">
+    <axis xyz="0 1 0" />
+    <origin xyz="0 0 0.2" rpy="0 0 0"/>
+    <parent link="base_link"/>
+    <child link="camera_link"/>
+  </joint>
+
+  <!-- Camera -->
```

Comment:
if you have this why do you need the static TF?

Reply:
This is so the point cloud is visualized correctly in rviz, see an example of the same problem [here.](https://answers.ros.org/question/329409/point-cloud-in-wrong-position-relative-to-robot/)
My understanding is the rviz frame does not necessarily line up with the camera's frame, so we publish an intermediate transform to make sure the visualization relative to the robot in rviz the same as it is in gazebo.

Reply:
oh god lol. Okay well in that case LGTM